### PR TITLE
Update build instructions after mill 0.9.7

### PR DESCRIPTION
**Pull request by rom1dep**
following https://github.com/scalafx/scalafx/issues/343#issuecomment-826300075

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,29 +49,33 @@ libraryDependencies ++= javaFXModules.map(m =>
...
+    Some((r: coursier.core.Resolution) =>
+       r.withOsInfo(coursier.core.Activation.Os.fromProperties(sys.props.toMap))
+    )
   }
 
-  // Add dependency on JavaFX libraries, OS dependent
-  val javaFXModules = List("base", "controls", "fxml", "graphics", "media", "swing", "web")
-    .map(m => ivy"org.openjfx:javafx-$m:16;classifier=$osName")
+  // Add dependency on JavaFX libraries
+  val javaFXVersion  = "16-ea+7"
```
Comment (jpsacha) [inline README.md:67]: This is referencing older early access version, can you change to regular release "16" in the rest if the README?

File: README.md [hunk 2]
```diff
@@ -49,29 +49,33 @@ libraryDependencies ++= javaFXModules.map(m =>
...
+  val scalaFXVersion = "16.0.0-R22"
+  val javaFXModules  = List("base", "controls", "fxml", "graphics", "media", "swing", "web")
+    .map(m => ivy"org.openjfx:javafx-$m:$javaFXVersion")
 
   def ivyDeps = {
     Agg(
-      ivy"org.scalafx::scalafx:16.0.0-R22"
+      ivy"org.scalafx::scalafx:$scalaFXVersion",
+      ivy"org.controlsfx:controlsfx:11.1.0",
```
Comment (jpsacha) [inline README.md:75]: controlsfx and not a needed dependency, can you remove this?

**Review by jpsacha — CHANGES_REQUESTED** [review]
Thanks for the update. I marked two small corrections

**Comment by rom1dep** [comment]
Should be good, now!

File: README.md (hunk 2 above)
Comment (rom1dep) [reply README.md]: I meant it as an example of how to fit other dependencies, but you're right. Removed it.

**Comment by jpsacha** [comment]
@rom1dep thanks for updating Mill setup

**Pull request merged**